hello.c

```diff
@@ -4,12 +4,13 @@ extern int returner();
 extern void asmprinter();
 extern int fib(int);
 
-void main() {
+int main() {
 	printf("hello, world\n");
 	printf("returner returned %d\n", returner());
 	asmprinter();
 
     for (int i = 1; i < 15; i++) {
         printf("fib(%d) = %d\n", i, fib(i));
     }
+    return 0;
```

Comment:
There was a curious side effect, the spike executable used to exit with code 14:

```
[...]
Makefile:37: recipe for target 'run-spike' failed
make: *** [run-spike] Error 14
```

Explicit `return 0` fixed this problem.